File: new_course/pattern/exm-3.js
//     *
//    **
//   ***
//  ****
// *****

const rows = 5;

for (let i = 1; i <= rows; i++) {
    //for printing spaces
    let starts = '';
    for (let j = 1; j <= rows - 1; j++) {
        starts += ' ';
    }

    //for printing start

    for (let j = 1; j <= i; j++) {
        starts += '*';
    }

    console.log(starts);
}

File: new_course/pattern/exm-3.cpp
//     *
//    **
//   ***
//  ****
// *****

#include <iostream>
using namespace std;

int main()
{
    int rows;
    cout << "Enter the number of rows: ";
    cin >> rows;

    for (int i = 1; i <= rows; ++i)
    {
        // for printing spaces
        for (int j = 1; j <= rows - i; ++j)
        {
            cout << " ";
        }

        // for printing starts spaces
        for (int k = 1; k <= i; ++k)
        {
            cout << "*";
        }
        cout << endl;
    }

    return 0;
}
